PrivacyCookie/DemoBundleBanner.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\PrivacyCookie;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\URLAliasService;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\URLAlias;
+use EzSystems\PrivacyCookieBundle\Banner\Banner;
+use EzSystems\DemoBundle\PrivacyCookie;
+use Symfony\Component\Translation\Translator;
+
+/**
+ * Builds a Privacy Cookie Banner object based on a configured Content Object.
+ */
+class DemoBundleBanner implements DemoBundleBannerFactory
+{
+    public static function build( Content $bannerContent, LocationService $locationService, URLAliasService $urlAliasService, Translator $translator )
+    {
+        $banner = new Banner();
+
+        $banner->caption = $bannerContent->getFieldValue( 'description' )->xml->textContent;
```

Comment:
not sure either, @bdunogier. If we let text for caption, maybe some user may thing in adding another image as caption... i will go for a textlinea or textarea. 

bonus topic for platform ui -> it would be awesome if we could define some kinda "rich text profiles" and devs could decide which rich text profile apply to a concrete field. i mean, maybe for a field you wan to allow bolds, italic but don't want bullets or images...